chainer/serializers/npz.py

```diff
@@ -0,0 +1,96 @@
+import numpy
+
+from chainer import cuda
+from chainer import serializer
+
+
+class DictionarySerializer(serializer.Serializer):
+
+    """Serializer for dictionary.
+
+    This is the standard serializer in Chainer. The chain hierarchy is simply
+    mapped to dictionary key.
+
+    Args:
+        dict (dict): The dict that this serializer represents.
+        path (str): The base path.
+
+    """
+    def __init__(self, dict, path=''):
+        self.dict = dict
+        self.path = path
+
+    def __getitem__(self, key):
+        return DictionarySerializer(self.dict, self.path + key + '/')
+
+    def __call__(self, key, value):
+        ret = value
+        if isinstance(value, cuda.ndarray):
```

Comment:
This check is required to accept Python scalar values (to_cpu only accepts numpy/cupy arrays).